base/permuteddimsarray.jl

```diff
@@ -77,11 +77,11 @@ end
 @inline genperm(I, perm::AbstractVector{Int}) = genperm(I, (perm...,))
 
 """
-    permutedims(A, perm)
+    permutedims(A::AbstractArray, [perm = (2,1)])
 
 Permute the dimensions of array `A`. `perm` is a vector specifying a permutation of length
-`ndims(A)`. This is a generalization of transpose for multi-dimensional arrays. Transpose is
-equivalent to `permutedims(A, [2,1])`.
+`ndims(A)`. This is a generalization of matrix transposition for multi-dimensional arrays.
+The value of `perm` defaults to `(2,1)` for transposing the elements of a matrix.
```

Comment:
"Transposing the elements of a matrix" makes it sound (at least to me) like it recursively calls `permutedims` on the elements. Perhaps "transposing the dimensions of a matrix"? Unless that isn't a standard way to say it.

Reply:
Perhaps "flips the array cross its diagonal" or "mirrors the array across its diagonal"?

Reply:
Yes, avoiding the word "transpose" would be ideal.